### ISSUE TITLE
Primary navigation

### ISSUE THREAD

**Issue by MirandaEcho** [INN/umbrella-citylimits#40]
Primary navigation bar styling.

**Comment by joshdarby** [comment]
@MirandaEcho Are we doing anything with the english/spanish language toggle on the new nav for now?

https://drive.google.com/drive/u/0/folders/1zxHL8M3XRY5YIfCjMCrhXFCRt6a332wV

**Comment by MirandaEcho** [comment]
@joshdarby - on the current site, the Espanol section links to the series that's in Spanish. We'll do the same with the toggle for now.

**Comment by joshdarby** [comment]
@MirandaEcho This is supposed to be sticky, right?

**Comment by joshdarby** [comment]
Needs:

- [ ] Desktop layout (sticky?)
- [ ] Mobile closed layout
- [ ] Mobile open layout

**Comment by joshdarby** [comment]
@MirandaEcho The first 3 categories on the mobile sticky nav are different than the first 3 categories in the desktop nav and the expanded mobile sticky nav. Is that a mistake or is that supposed to be a separate menu?

For reference:

Desktop:
![Screen Shot 2019-10-15 at 10 35 09 AM](https://user-images.githubusercontent.com/18353636/66841382-9d21f980-ef37-11e9-8cda-21a3513651b1.png)

Mobile expanded:
![Screen Shot 2019-10-15 at 10 36 25 AM](https://user-images.githubusercontent.com/18353636/66841424-af039c80-ef37-11e9-8ddd-65b68105e2e1.png)


Mobile sticky:
![Screen Shot 2019-10-15 at 10 34 57 AM](https://user-images.githubusercontent.com/18353636/66841383-9d21f980-ef37-11e9-91ce-dcd8492d6019.png)

**Comment by MirandaEcho** [comment]
@joshdarby - it is a separate mobile nav.

For the sticky on desktop question, I'm waiting on final confirmation from Jarrett but leaning not sticky.

**Comment by kaylima** [comment]
@miranda was there final confirmation from Jarrett that the nav will not be sticky?

**Comment by MirandaEcho** [comment]
Not sticky.

**Comment by joshdarby** [comment]
@MirandaEcho @kaylima I thought we decided that it was supposed to be sticky and pick up after it was scrolled past on mobile? That was implemented in https://github.com/INN/umbrella-citylimits/pull/65. It might not work on staging right now because of https://github.com/INN/umbrella-citylimits/issues/102#issuecomment-552482726

**Comment by MirandaEcho** [comment]
@joshdarby - if its already done, great!
